src/services/dashboards/dashboard-detail/DashboardDetailPage.vue

```diff
@@ -181,6 +181,7 @@ const handleVisibleNameEditModal = () => {
 };
 const handleNameUpdate = (name: string) => {
     dashboardDetailState.name = name;
+    dashboardDetailOriginState.dashboardInfo.name = name;
```

Comment:
question: I thought origin state must be updated only by api.. Is this the only way?